src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work ⚙.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped its work.");

```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should also be more informative and indicate that it's a Truck that has stopped working. For instance: `Truck has stopped its work`.